app/templates/gentelella/users/events/wizard/basic-details.html

```diff
@@ -212,10 +212,9 @@
                 <transition name="fade">
                     <span class="help-block" v-show="!isValidLinkEntry(event.event_url)">This looks like an invalid URL and hence will be ignored.</span>
                 </transition>
-                <span class="input-group-btn" v-bind:style="{ opacity: 1 - event.social_links.length  }">
+                <span class="input-group-btn" v-bind:style="{ opacity: 1 }">
```

Comment:
`v-bind:style="{ opacity: 1 }"` not needed unless we plan to modify the opacity based on anything. It is anyway at opacity 1 by default